exp-models/addon/models/experiment.js

```diff
@@ -25,11 +25,18 @@ export default DS.Model.extend(JamModel, {
     structure: DS.attr(),
     displayFullscreen: DS.attr('boolean'),
 
+    duration: DS.attr('string'),
```

Comment:
All of these newly added fields should be added to the experiment definition in `generate_schemas` before we forget.